ssz-rs/src/uint.rs

```diff
@@ -45,7 +44,9 @@ macro_rules! define_uint {
         impl Merkleized for $uint {
             fn hash_tree_root(&mut self) -> Result<Node, MerkleizationError> {
                 let mut root = vec![];
-                let _ = self.serialize(&mut root)?;
+                let _ = self
+                    .serialize(&mut root)
+                    .map_err(|_| MerkleizationError::SerializationError);
```

Comment:
same thing here w/ dropping the error checking

-- I see we do this because we lose the auto-derive `From` impl -- I'd actually prefer to keep it if we can and while it means adding more code manually we don't have to worry about correctness anywhere else -- can you add the `From` impls?

Reply:
Totally, much cleaner. Addressed in https://github.com/ralexstokes/ssz-rs/pull/25/commits/b58448c30caa3cfc15bf696c020abb01001cac89.